website/docs/misc/i18next.md

```diff
@@ -45,48 +51,72 @@ module.exports = {
 ```
 
 ```js
-import { i18n } from "@lingui/core";
+import { t } from "@lingui/core/macro";
 
-document.getElementById("output").innerHTML = i18n.t({ id: "key", message: "Hello world" });
+document.getElementById("output").innerHTML = t`Hello world`;
 ```
 
-:::note
-The `message` property can be specified in the case of [Message Extraction](/docs/guides/message-extraction.md) usage flow. You can use the `i18n.t` function only with the `id`, but in this case you'll have to manage your localization catalogs yourself, without advantages of the [message extraction](/docs/guides/message-extraction.md) feature.
+:::tip
+This example uses a macro for the translation. Macros are a powerful feature of Lingui that allows you to write messages directly in your code. Read more about [Macros](/docs/ref/macro.mdx).
 :::
 
+If you prefer to use explicit IDs for your messages, you can utilize the `msg` macro:
+
+```js
+import { msg } from "@lingui/core/macro";
+
+msg({ id: "msg.greeting", message: `Hello World` });
+```
+
+Read more about [Explicit vs Generated Message IDs](/docs/guides/explicit-vs-generated-ids.md).
```

Comment:
It also possible tow write explicit id with the `t` macro, 

```js
import { t } from "@lingui/core/macro";

document.getElementById("output").innerHTML = t({ id: "msg.greeting", message: `Hello World` });
```

The difference between `msg` and `t` is first one returns `MessageDescriptor`, second one wraps this message descriptor into global `i18n._` method, so returns a translated string directly. The signature of these to macros is almost the same.